ui/src/org/pentaho/di/ui/job/entries/ftpdelete/JobEntryFTPDeleteDialog.java

```diff
@@ -1307,6 +1307,10 @@ private boolean connectToFTPS() {
       }
       retval = true;
     } catch ( Exception e ) {
+      if ( ftpsclient != null ) {
+        ftpsclient.disconnect();
```

Comment:
I believe this should be:

if ( ftpsclient != null ) {
  try {
    ftpsclient.disconnect();
  } catch ( Exception ignored ) {
    // Cannot do anything else here
  }
  ftpsclient = null;